src/data/faq_de.json

```diff
@@ -1887,6 +1887,67 @@
+                    {
+                        "title": "Warum erhalte ich keine Risikowarnung, obwohl ich Kontakt zu einer Person hatte, die eine Risikobegegnung in ihrer App angezeigt bekommt? ",
+                        "anchor": "risk_encounter_no_warning",
+                        "active": false,
+                        "textblock": [
+                            "Risikowarnungen werden ausschließlich durch das Teilen von positiven Testergebnissen über die Corona-Warn-App ermöglicht. Die Anzeige einer Risikobegegnung auf einem Gerät führt nicht zu einer Risikowarnung auf Geräten von anderen Nutzern. Nur wenn ein Nutzer auch ein positives Testergebnis über die CWA teilt, kann eine Warnung für dessen Kontakte erfolgen.",
+                            "Siehe auch FAQ: <ul><li><a href='#risk_encounter_not_realtime' target='_blank'>Warum erhalte ich erst nach mehreren Tagen nach der Risikobegegnung eine Risikowarnung?</a></li><li><a href='#risk_encounter_different_devices' target='_blank'>Wie kommt es zu Unterschieden in der Risikobewertung auf mehreren Geräten, auch wenn die jeweiligen Smartphones sich nicht weit voneinander entfernt befanden?</a></li><li><a href='#pos_test_no_risk_encounter' target='_blank'>Ich wurde positiv getestet. Warum erhält eine Person im gleichen Haushalt keine Risikowarnung?</a></li><li><a href='#risk_encounter_doubted' target='_blank'>Warum erhalte ich eine Risikowarnung für ein Datum, an dem ich keine Kontakte hatte / nicht das Haus verlassen habe?</a></li><li><a href='#risk_encounter_place' target='_blank'>Werden zu einer Risiko-Begegnung Personen und/oder Orte zugeordnet?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "Wie kommt es zu Unterschieden in der Risikobewertung auf mehreren Geräten, auch wenn die jeweiligen Smartphones sich nicht weit voneinander entfernt befanden?",
+                        "anchor": "risk_encounter_different_devices",
+                        "active": false,
+                        "textblock": [
+                            "Abweichende Risikobewertungen können mehrere Ursachen haben, beeinträchtigen aber nicht die Wirksamkeit der Corona-Warn-App.",
+                            "Die Einschätzung eines möglichen Infektionsrisikos hängt von ermittelter Kontaktdauer und Abstand zum Smartphone eines anderen CWA-Nutzers ab, der sich dazu entschlossen hat, sein positives Testergebnis zu teilen. Die Ermittlung des Abstandes wird dabei auch von der Ausrichtung der beiden Smartphone-Antennen zueinander beeinflusst. Daher ist das Ergebnis der Risikoberechnung auch abhängig von der genauen räumlichen Lage und Ausrichtung der beteiligten Smartphones. Auch die Umgebung (reflektierende oder absorbierende Oberflächen) können die Reichweite des BLE-Signals beeinflussen, sodass je nach Ausrichtung der Smartphones unterschiedliche Werte für Kontaktdauer und Signaldämpfung (Abstand) empfangen werden können.",

```

Comment:
The abbreviation "BLE" in "BLE-Signal" is not explained. Maybe this is a candidate for a new entry in the Glossary?

Reply:
A glossary entry will follow soon

Reply:
PR: https://github.com/corona-warn-app/cwa-website/pull/2534